stablehlo/reference/Element.cpp

```diff
@@ -615,113 +615,55 @@ Element complex(const Element &e1, const Element &e2) {
 
 Element convert(Type type, bool value) {
   if (isSupportedBooleanType(type)) return Element(type, value);
+  return convert(type, value ? 1UL : 0UL);
+}
+
+Element convert(Type type, APSInt value) {
```

Comment:
We may delay pervasive adoption of APSInt until later, but I think we should adopt it in this specific helper function to deduplicate.